charts/generic-prometheus-alerts/templates/_helpers.tpl

```diff
@@ -50,3 +50,25 @@ Selector labels
 app.kubernetes.io/name: {{ include "generic-prometheus-alerts.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
 {{- end }}
+
+{{/*
+Tags for the grafana dashboards
+*/}}
+{{- define "generic-prometheus-alerts.dashboardTags" -}}
+{{- .Release.Name | append .Values.extraDashboardTags | toJson -}}
+{{- end }}
+
+{{/*
+Labels for the grafana dashboard configmaps
+*/}}
+{{- define "generic-prometheus-alerts.dashboardLabels" -}}
+grafana_dashboard: ""
```

Comment:
does this need a value?

Reply:
Yeah unfortunately it looks like it doesn't get slurped into grafana without it 😞

Reply:
Ah, sorry, just realised I didn't answer your question - nope, it doesn't need a value, but it does need to be there in order for the loading to happen.

Reply:
yes, I just wondered if it was safer to set a value, even it's not used.

Reply:
I've not tried setting a value as yet - will have an experiment later to see if it makes a difference 👍